lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3513,10 +3509,20 @@ class PropertyEmitter : public EmitterBase,
 };
 } // end anonymous namespace
 
-void PropertyEmitter::emitProperty(
-    Value property, PropertyPrecedence parenthesizeIfLooserThan) {
+void PropertyEmitter::emitEnabledProperty(
+    Value property, Value enable, PropertyPrecedence parenthesizeIfLooserThan) {
   assert(localTokens.empty());
-  // Wrap to this column.
+
+  // If the property is tied to an enable, emit that.
+  if (enable) {
+    ps << "disable iff" << PP::nbsp << "(~(";
```

Comment:
This negation doesn't run into any `x` or `z` issues?

Reply:
not syntactically, but behaviorally maybe, I guess that if an X or Z is encounter, we don't really give any guarantees on the behavior anyway no?